### PR TITLE
add notes about snapcraft usage

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

File: docs/snapcraft-usage.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Instead of sideloading your app manually, you can use the following to
+speed up things:
+
+	$ snapcraft run
+
+Run this inside your project and an Ubuntu Core VM will be booted, your last
+successful .snap build will be uploaded to it and you will be connected to it
+via SSH automatically. Type `Ctrl-D` at end of testing your uploaded app and
+your back to your previous working directory.
```
Comment (elopio) [inline docs/snapcraft-usage.md]: *you are*

**Comment by elopio** [comment]
Thanks Daniel. I left a typo comment
I'm not sure about documenting run. We didn't give it a lot of love and Sergio already has a branch removing it. @sergiusens, please comment.

**Comment by dholbach** [comment]
Hohum... I always thought that 'snapcraft run' was a great feature... do we have anything replacing it or anything else we suggest?

**Comment by sergiusens** [comment]
@elopio and @dholbach 
`snapcraft run` is being replaced with `snappy try`

More importantly we need to split the docs in 15.04 and 16.04

**Comment by sergiusens** [comment]
fwiw, it is staying for 15.04, that means `trusty`, `vivid` and `wily`; just gone from `xenial`

**Pull request merged**